source/features/hide-in-case-you-missed-notifications.js

```diff
@@ -0,0 +1,5 @@
+export default function () {
+	$('.with-contextTweet')
+		.filter((index, tweet) => tweet.textContent.toLowerCase().includes('in case you missed'))
```

Comment:
This will not work correctly for non-English Twitter users.

Reply:
Apologies. I originally thought there was no unique selector for "In case you missed" notifications, but it looks like `[data-component-context="generic_activity_MagicRecFirstDegreeTweetRecap"]` will do it! Let me know if there are any further changes.

Reply:
I see the CI build failed, but I'm not seeing those same errors when I test locally. Not sure how to proceed.

Reply:
> I see the CI build failed, but I'm not seeing those same errors when I test locally. Not sure how to proceed.

That's fixed in master.